tests/test_wafv2/test_wafv2.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import unicode_literals
+import json
+import pytest
+
+import sure  # noqa
+import boto3
+from botocore.exceptions import ClientError
+from moto import mock_wafv2
+from test_helper_functions import CREATE_WEB_ACL_BODY, LIST_WEB_ACL_BODY
```

Comment:
Looks like our CI is falling over this - it should probably be a relative import:
```
from .test_helper_functions import CREATE_WEB_ACL_BODY, LIST_WEB_ACL_BODY
```

Reply:
Thanks for all the help! I would like to add associate-web-acl and dissociate-web-acl is this okay?
https://docs.aws.amazon.com/waf/latest/APIReference/API_AssociateWebACL.html
https://docs.aws.amazon.com/waf/latest/APIReference/API_DisassociateWebACL.html

Reply:
> 
> 
> Thanks for all the help! I would like to add associate-web-acl and dissociate-web-acl next. Is this okay?
> https://docs.aws.amazon.com/waf/latest/APIReference/API_AssociateWebACL.html
> https://docs.aws.amazon.com/waf/latest/APIReference/API_DisassociateWebACL.html

Definitely - more PR's are always welcome 🙂